openff/evaluator/protocols/forcefield.py

```diff
@@ -1036,7 +1042,9 @@ def _generate_charges(self, molecule):
             raise ValueError("Invalid toolkit specification.")
 
         molecule.generate_conformers(toolkit_registry=toolkit_wrapper)
-        molecule.compute_partial_charges_am1bcc(toolkit_registry=toolkit_wrapper)
+        molecule.assign_partial_charges(partial_charge_method='am1bcc',
+                                        toolkit_registry=toolkit_wrapper,
+                                        normalize_partial_charges=True)
```

Comment:
Nice catch, could you apply a similar change in `openff/evaluator/protocols/coordinates.py`?

```diff
diff --git a/openff/evaluator/protocols/coordinates.py b/openff/evaluator/protocols/coordinates.py
index 59d35aa..a7055ca 100644
--- a/openff/evaluator/protocols/coordinates.py
+++ b/openff/evaluator/protocols/coordinates.py
@@ -438,7 +438,7 @@ class BuildDockedCoordinates(Protocol):
         # Assign AM1-BCC charges to the ligand just as an initial guess
         # for docking. In future, we may want to get the charge model
         # directly from the force field.
-        ligand.compute_partial_charges_am1bcc()
+        ligand.assign_partial_charges(partial_charge_method="am1bcc")
 
         return ligand.to_openeye()
 
diff --git a/openff/evaluator/protocols/forcefield.py b/openff/evaluator/protocols/forcefield.py
index 7cb3488..3f07600 100644
--- a/openff/evaluator/protocols/forcefield.py
+++ b/openff/evaluator/protocols/forcefield.py
@@ -1036,7 +1036,10 @@ class BuildTLeapSystem(TemplateBuildSystem):
             raise ValueError("Invalid toolkit specification.")
 
         molecule.generate_conformers(toolkit_registry=toolkit_wrapper)
-        molecule.compute_partial_charges_am1bcc(toolkit_registry=toolkit_wrapper)
+        molecule.assign_partial_charges(
+            partial_charge_method="am1bcc",
+            toolkit_registry=toolkit_wrapper,
+        )
 
     def _parameterize_molecule(self, molecule, force_field_source, cutoff):
         """Parameterize the specified molecule.
```
Partial charges are normalized by default from the toolkit so `normalize_partial_charges=True` is not necessary unless there's some nuance with `tleap` I'm unaware of

Reply:
tleap imposes a check of molecule (tleap calls it a residue) total net partial charge and raises warnings if the total net charge is not integral. 

and yes, `normalize_partial_charges=True` is the default. I agree it is unnecessary. I just added it to remind myself about this nuance and just in case if the default of openff-toolkit changes in the future, since the openff-toolkit API is not quite stable yet.

Reply:
I'm okay leaving it in if you prefer it there; it is unlikely that the toolkit will change the default to `False` but it's fine to include it to match the behavior of `tleap`